proposals/KEP-0002-github-actions-workflow-alignment/README.md

```
@@ -0,0 +1,163 @@
+
+1. Use `docker/build-push-action` for multi-arch builds (linux/arm64, linux/amd64)
+2. Generate metadata with `docker/metadata-action` for tags and labels
+3. Sign images using cosign with digest-based references (e.g., `image@sha256:...`)
+4. Generate SPDX SBOMs with `anchore/sbom-action`
+5. Attest (not attach) SBOMs using cosign to ensure signature with `cosign attest`
+
+### Risks and Mitigations
+
+**Risk**: Workflow changes may temporarily break image builds.
+
+**Mitigation**: Test the workflow changes as possible in the PR created for this KEP before merging. Workflows already exist in PR #1790](https://github.com/kubeflow/model-registry/pull/1790#pullrequestreview-3374876556) and [PR #1588](https://github.com/kubeflow/model-registry/pull/1588) as partial implementations. Merge the PR for this KEP right after a release, so to mitigate potential unforeseen issues while integrating this strategy for all workflows (i.e.: not jeopardize the release process).
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)